homeassistant/components/media_player/sonos.py

```diff
@@ -42,6 +42,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     players = None
     hosts = config.get('hosts', None)
+    group_devices = config.get('group_devices', False)
```

Comment:
For the sake of documentation, this is how `groups` looks like on my system:

```python
{'RINCON_000E5876C00A01400:63': ZoneGroup(uid='RINCON_000E5876C00A01400:63', coordinator=SoCo("192.168.1.112"), members={SoCo("192.168.1.92"), SoCo("192.168.1.247"), SoCo("192.168.1.112")}),
 'RINCON_000E5876C00A01400:64': ZoneGroup(uid='RINCON_000E5876C00A01400:64', coordinator=SoCo("192.168.1.204"), members={SoCo("192.168.1.204")})}
```

`192.168.1.204` is the player in the bathroom, the other 3 are Living Room (L + R) and Kitchen.